### DISCUSSION TITLE
成果清单中的英文姓名如何显示全称，目前姓的部分只显示首字母

### DISCUSSION THREAD

**Discussion by li-aolong** [BITNP/BIThesis#562] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/fcd2dcce-1962-428e-8ada-a99ebb666f03)

**Comment by YDX-2147483647** [comment, marked as answer]
可以编辑`main.tex`开头的`\usepackage[…]{biblatex}`：

```diff
- gbnamefmt=lowercase,
+ gbnamefmt=quanpin,
```

![图片](https://github.com/user-attachments/assets/4457e062-da30-4b4d-a7c3-670407291d1a)

这会连正文参考文献列表一起改；如果只想改清单，我再查查。

<details>
<summary>该选项具体含义</summary>

[请参考 biblatex-gb7714-2015 手册](http://mirrors.ctan.org/macros/latex/contrib/biblatex-contrib/biblatex-gb7714-2015/biblatex-gb7714-2015.pdf)：

![图片](https://github.com/user-attachments/assets/f0fdbd5c-f51f-44cb-8476-8cadd6b1bf31)

</details>

<details>
<summary>其实“ZHANG S”这样只留首字母也符合国标要求</summary>

[GB/T 7714—2015](https://lib.tsinghua.edu.cn/wj/GBT7714-2015.pdf)：

![图片](https://github.com/user-attachments/assets/28e62fd2-8cf1-431b-82fa-9f3a34a16966)

陈浩元（国标主要起草人之一）在《编辑学报》上发表的《GB/T 7714新标准对旧标准的主要修改及实施要点提示》：

![图片](https://github.com/user-attachments/assets/d6e3e2d3-83f1-473b-8577-12b9a870eb5a)

另可参考 #401 开头给出的严格符合国标的设置。

</details>
